climada/hazard/wildfire.py

```diff
@@ -0,0 +1,1165 @@
+        res_centr = centroids._centroids_resolution(centroids)
+
+        # fire identification
+        while firms.iter_ev.any():
+            # Compute cons_id: consecutive fires in current iteration
+            self._firms_cons_days(firms)
+            # Compute clus_id: cluster identifier inside cons_id
+            self._firms_clustering(firms, res_data, self.clus_thres_firms)
+            # compute event_id
+            self._firms_fire(self.days_thres_firms, firms.cons_id.values, \
+                             firms.clus_id.values, firms.event_id.values, \
+                             firms.iter_ev.values, firms.datenum.values)
+            LOGGER.info('Remaining fires to identify: %s.', str(np.argwhere(\
+            firms.iter_ev.values).size))
+
+        # remove minor fires
+        if self.remove_minor_fires_firms:
+            firms = self._firms_remove_minor_fires(firms,
+                                                   self.minor_fire_thres_firms)
+
+        # compute brightness and fill class attributes
+        LOGGER.info('Computing intensity of %s fires.',
+                    np.unique(firms.event_id).size)
+        self._calc_brightness(firms, centroids, res_centr)
+
+
+    def hull_burned_area(self, ev_id, alpha=100.87, return_plot=False):
+        """Compute the burned area for a given fire.
+        Please note: NASA advises against calculating burned area using the
+        FIRMS data.
+
+        Algorithm used: https://pypi.org/project/alphashape/
+
+        Parameters:
+            ev_id: id of the selected fire
+            alpha (float, optional): parameter used to compute the concave hull
+            return_plot (bool, optional): indicate if the output plot of the
+                concave hull algorithm should be returned
+            info
+
+        Returns:
+            float
+        """
+        ev_idx = np.argwhere(self.event_id == ev_id).reshape(-1)[0]
+        if not ev_idx.size:
+            LOGGER.error('No event with id %s', str(ev_id))
+            raise ValueError
+
+        if not self.centroids.lat.size:
+            self.centroids.set_meta_to_lat_lon()
+
+        # Extract coordinates where intensity > 0 (for a given fire)
+        fire_lat = self.centroids.lat[self.intensity[ev_idx, :].nonzero()[1]]
+        fire_lon = self.centroids.lon[self.intensity[ev_idx, :].nonzero()[1]]
+
+        # Creation of the geodataframe
+        fire = gpd.GeoDataFrame(crs=DEF_CRS)
+        fire['geometry'] = list(zip(fire_lon, fire_lat))
+        fire['geometry'] = fire['geometry'].apply(Point)
+        points = fire.geometry.values
+
+        # Compute concave hull
+        concave_hull, _ = alpha_shape(points, alpha=alpha)
+
+        # Compute area concave hull in right projection
+        project = partial(
+            pyproj.transform,
+            pyproj.Proj(init=DEF_CRS['init']), # source coordinate system
+            pyproj.Proj(init='epsg:3310')) # destination coordinate system: albers california
```

Comment:
Why epsg:3310? (This is pure ignorance) But it looks like a locally optimized projected crs for the US and some regions in Canada

Reply:
Code from Gabriela. I don't know either =)

They focused largely on California, maybe that's why

Reply:
But this module is not made for California only? Then please adjust.

Reply:
I've removed the alpha_shape and the hull_bruned_area method in commit 13a2c8c as calculating area burned is not used and the module is complex already